includes/diagnostics.php

```diff
@@ -117,6 +117,10 @@
     }
 }
 
+if ( defined( 'WP_REDIS_SERVERS' ) ) {
+    $info['WP_REDIS_SERVERS'] = preg_replace('/password=(.+[a-zA-Z])(?=&{0,1})/um', 'password=********', $info['WP_REDIS_SERVERS']);
```

Comment:
Can we make this a helper method, so we don't duplicate the logic?

Do any other constants need to be checked?